### PR TITLE
Create news item

### LINKED ISSUE #7: Complete the News model
See the screenshot below.

Screen Shot 2563-10-09 at 09 56 21

We should be able to save a news with either the title and URL or text.

### PR DESCRIPTION
**Pull request by liisberg**
Fixes #7.
I have not added any UI, as I am not as expert on this. 
Perhaps would be a good beginner issue? :)

### CODE REVIEW

**Comment by zkan** [comment]
@liisberg Thanks, looking great!

For the `create_item` view, could you create a separate PR for it, please? It will be better if a PR has only one objective. :)

**Comment by liisberg** [comment]
> @liisberg Thanks, looking great!
> 
> For the `create_item` view, could you create a separate PR for it, please? It will be better if a PR has only one objective. :)

Sure thing! Requested changes have been made.

**Comment by liisberg** [comment]
@zkan Please review this PR when you have the chance :-)

File: dataengnews/news/views.py [hunk 1]
```diff
@@ -1,3 +1,10 @@
 from django.shortcuts import render
```
Comment (zkan) [inline dataengnews/news/views.py:1]: Looks like `render` is not used any where in this file. Please remove it.

**Review by zkan — COMMENTED** [review]
@liisberg Just one small request change. :)

File: dataengnews/news/views.py (hunk 1 above)
Comment (liisberg) [reply dataengnews/news/views.py]: Done! :-)

**Comment by zkan** [comment]
@liisberg Thanks for your contribution!

**Pull request merged**